{
  "contest": 304,
  "handle": "fakesm1le",
  "risk": "2",
  "title": "Mishandling error from database save in observeInTX() might result in double processing of blocks",
  "issueId": 455,
  "issueUrl": "https://github.com/code-423n4/2023-11-zetachain-findings/issues/455"
}